src/layout/download/furnace.js

```diff
@@ -61,7 +61,6 @@ const Furnace = ({ components, _ID, _body, _parseYaml, _relativeURL }) => {
 								Object.keys( MODULES )
 									.map( ( module, i ) => (
 										<li className="furnace__component" key={ i }>
```

Comment:
```suggestion
										<li className="furnace__component" key={ i }>

```

Reply:
It's a bit easier to read if we preserve newlines, makes the change diff smaller :)